src/tools/genpolicy/rules.rego

```diff
@@ -1319,6 +1319,31 @@ ExecProcessRequest {
     print("ExecProcessRequest 3: true")
 }
 
+UpdateRoutesRequest {
```

Comment:
Please try to make these rules shorter. On substance, the rules look reasonable to me.